drivers/timer/arcv2_timer0.c

```diff
@@ -322,7 +331,8 @@ void z_clock_set_timeout(s32_t ticks, bool idle)
 	ticks = MIN(MAX_TICKS, ticks);
 
 	/* Desired delay in the future */
-	delay = (ticks == 0) ? CYC_PER_TICK : ticks * CYC_PER_TICK;
+	delay = ticks * CYC_PER_TICK;
```

Comment:
Same here - it's a good move to use much more readable `MAX` macro but especially given we change a functionality here a little bit (move from `CYC_PER_TICK` to `MIN_DELAY` in one case) I'd prefer to have it as a separate commit with clear explanation on why we can do such a transition. From visible diff context here and existing commit message I cannot understand why that change is possible at all and if it doesn't break anything.

Reply:
already changed